modules/RTC/RTC.js

```diff
@@ -84,9 +84,10 @@ RTC.obtainAudioAndVideoPermissions = function (options) {
 };
 
 RTC.prototype.onIncommingCall = function(event) {
-    if(this.options.config.openSctp)
+    if(this.options.config.openSctp && !RTCBrowserType.usesORTC()) {
```

Comment:
If the executing browser uses ORTC what? Something's missing? Something doesn't work? I expect a comment explaining the exclusion to come handy.

Can we detect it by feature instead of by checking the browser type?